linux/LinuxProcessList.c

```diff
@@ -1340,9 +1347,28 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_
       if (settings->flags & PROCESS_FLAG_IO)
          LinuxProcessList_readIoFile(lp, procFd, pl->realtimeMs);
 
-      if (!LinuxProcessList_readStatmFile(lp, procFd, !!(settings->flags & PROCESS_FLAG_LINUX_LRS_FIX), pl->realtimeMs))
+      if (!LinuxProcessList_readStatmFile(lp, procFd))
          goto errorReadingProcess;
 
+      {
+         bool prev = proc->usesDeletedLib;
+         proc->usesDeletedLib = false;
```

Comment:
Only reset this flag, once we actually re-calculate it. Maybe put directly before call to `LinuxProcessList_readMaps`?

Reply:
true, thanks for catching

Reply:
I think that's why we do those code reviews ;-) Less stuff we need to fix later. :P